hieradata/org/lsst/role/hexrot.yaml

```diff
@@ -5,7 +5,7 @@ classes:
   - "profile::core::ni_packages"
   - "docker"
   - "python"
-  - "x2go"
+  - "profile::core::x2go_agent"
```

Comment:
I'd prefer that we use `profile::core::common::install_x2go: true` or directly include the `profile::core::x2go_agent` class but try to avoid using two idioms that accomplish the same result.